include.js

```diff
@@ -1,3 +1,3 @@
 var why = require('./')
 
-process.on('SIGUSR1', function() { why() })
+require('siginfo')(why)
```

Comment:
you should pass the force flag here for semver minor update

Reply:
done